Hotrod/main.py

```diff
@@ -0,0 +1,278 @@
+        when a new game or level is started.
+        """
+
+        for enemy in self.game.enemies:
+            enemy.reset_scatter_timers()
+            enemy.reset_release_timers()
+
+    def update_play_area(self, instance, value):
+        """Kivy event triggered when level size changes to ensure that all
+        elements of the play area are positioned and sized correctly"""
+        for column in self.game.level.cells:
+            for cell in column:
+                cell.update_cell()
+
+        for enemy in self.game.enemies:
+            enemy.update_character()
+        self.game.player.update_character()
+
+    def reset_after_death(self):
+        """Reset the characters' positions and reset the scatter timer.
+
+        This method both resets the characters' positions as well
+        as resetting the enemies' scatter mode change timers.
+        It should only be called when the player has lost a life
+        without it resulting in a game over."""
+
+        self.initialise_characters()
+        for enemy in self.game.enemies:
+            enemy.reset_scatter_timers()
+
+
+class HotrodGame(Widget):
+    """Manage the game and application.
+
+    This widget manages and controls the application.
+    This widget has access to all major gameplay widgets, in addition
+    to general game properties such as score and lives.
+    Gameplay widgets access each other through this widget.
+    """
+    # Reference to the play area
+    play_area = ObjectProperty(None)
+    # Reference to the level
+    level = ObjectProperty(None)
+    # Reference to the player
+    player = ObjectProperty(None)
+    # List containing references to the enemies
+    enemies = ListProperty()
+
+    score = NumericProperty(INITIAL_SCORE)
+    lives = NumericProperty(INITIAL_LIVES)
+
+    pellet_value = NumericProperty(10)
+    powerup_length = NumericProperty(10)
+
+    # GUI elements so that they can be referred to in
+    # multiple methods
+    game_over_screen = ObjectProperty(None)
+    heads_up_display = ObjectProperty(None)
+
+    # Dictionary containing all sounds used in the game
+    sounds = ObjectProperty()
+
+    def start(self):
+        self.play_area.start_game()
+
+    def load_sounds(self):
+        """Load the sounds used in the game.
+
+        This method loads the sounds for use in the game and
+        stores them in a dictionary for easy access.
+        """
+        self.sounds = {}
+        self.sounds['bg'] = SoundLoader.load(os.path.join(SOUND_DIRECTORY, 'song.wav'))
+        self.sounds['jingle'] = SoundLoader.load(os.path.join(SOUND_DIRECTORY, 'jingle.wav'))
+        self.sounds['chomp_high'] = SoundLoader.load(os.path.join(SOUND_DIRECTORY, 'chomp_high.wav'))
+        self.sounds['chomp_low'] = SoundLoader.load(os.path.join(SOUND_DIRECTORY, 'chomp_low.wav'))
+        self.sounds['retreat'] = SoundLoader.load(os.path.join(SOUND_DIRECTORY, 'retreat.wav'))
+
+    def update(self, dt):
```

Comment:
Another small thing I was indecisive about... 
Couldn't decide whether the PlayArea (which handles things directly related to the visual representation of the game) should be in charge of updating movement, rather than the HotrodGame (which has references to all gameplay elements and handles game start/stop, general properties, user input and user interface)... 
After writing this, it sounds like it's more suited to the PlayArea.